.github/workflows/lock-versions.yml

```diff
@@ -6,7 +6,7 @@ on:
           description: 'List of branches to lock versions (ordered, comma separated)'
           required: true
           # 7.17 was intentionally skipped because it was added late and was bug fix only
-          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9'
+          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9,8.10'
```

Comment:
time to drop?

Reply:
@Mikaayenson - dropped support so it is only 8.7 -> 8.11. This required us to trim the version lock file, the lock versions workflow and of course the stack schema map. I also had to fix a bug in New Terms `validate()` method so that the current package version took priority over the min-stack in the rule for ECS validation.